### PR TITLE
various documentation updates

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

File: docs/PORTABLE_SERVICES.md [hunk 1]
```diff
@@ -153,9 +153,9 @@ images should just work the way they are. Specifically, the following
 requirements are made for an image that can be attached/detached with
 `portablectl`.
 
-1. It must contain a binary (and its dependencies) that shall be invoked,
-   including all its dependencies. If binary code, the code needs to be
-   compiled for an architecture compatible with the host.
+1. It must contain am executable that shall be invoked, along with all its
```
Comment (dnicolodi) [inline docs/PORTABLE_SERVICES.md]: Typo `s/am/an/`

**Comment by poettering** [comment]
Force pushed a new version with the typo @dnicolodi discovered fixed. ptal!

File: docs/PORTABLE_SERVICES.md [hunk 2]
```diff
@@ -177,15 +177,35 @@ requirements are made for an image that can be attached/detached with
...
+or `mkosi` generally qualify for all of the above in one way or another. If you
+wonder what the most minimal image would be that complies with the requirements
+above, it could consist of this:
 
 ```
 /usr/bin/minimald                        # a statically compiled binary
 /usr/lib/systemd/minimal-test.service    # the unit file for the service, with ExecStart=/usr/bin/minimald
 /usr/lib/os-release                      # an os-release file explaining what this is
+/etc/resolv.conf                         # empty file to mount over with host's version
+/etc/machine-id                          # dito
```
Comment (keszybz) [inline docs/PORTABLE_SERVICES.md]: ditto

File: docs/PORTABLE_SERVICES.md [hunk 3]
```diff
@@ -177,15 +177,35 @@ requirements are made for an image that can be attached/detached with
...
+/proc/                                   # empty directory to use as mount point for host's API fs
```
Comment (keszybz) [inline docs/PORTABLE_SERVICES.md:203]: ditto ditto

File: docs/PORTABLE_SERVICES.md [hunk 4]
```diff
@@ -195,6 +215,11 @@ own. If they do, it's fine, it will be ignored by the portable service logic,
 but they generally don't have to, and it might make sense to avoid any, to keep
 images minimal.
 
+If the image is writable, and some of the files or directories that are
+overmounted from the host do not exist they are created if needed. On
+read-only, immutable images (e.g. squashfs images) all files and directories to
+over-mount must be pre-existing however.
```
Comment (keszybz) [inline docs/PORTABLE_SERVICES.md]: "must exist already"?

File: man/systemd.kill.xml [hunk 5]
```diff
@@ -87,19 +87,15 @@
...
-        signal specified via <varname>FinalKillSignal=</varname> (unless
-        this is disabled via the <varname>SendSIGKILL=</varname>
-        option). See
-        <citerefentry><refentrytitle>kill</refentrytitle><manvolnum>2</manvolnum></citerefentry>
-        for more information.</para>
+        <para>Processes will first be terminated via <constant>SIGTERM</constant> (unless the signal to send
+        is changed via <varname>KillSignal=</varname>). Optionally, this is immediately followed by a
+        <constant>SIGHUP</constant> (if enabled with <varname>SendSIGHUP=</varname>). If the main process of
+        a unit then exits or if after a delay (configured via the <varname>TimeoutStopSec=</varname> option),
+        processes still remain, the termination request is repeated with the <constant>SIGKILL</constant>
```
Comment (keszybz) [inline man/systemd.kill.xml]: I'd put the commas in a different place:
"If processes still remain after the main process of a unit has exited or the delay configured via the `<varname>TimeoutStopSec=</varname>` has passed, the termination request..."

File: docs/PORTABLE_SERVICES.md (hunk 2 above)
Comment (poettering) [reply docs/PORTABLE_SERVICES.md]: interesting, German and English appear to have imported an Italian word, but adopted different spelling for it. frickin' confusing

Comment (keszybz) [reply docs/PORTABLE_SERVICES.md]: Not really ;)
> First attested in 1625. From regional Italian ditto, variant of detto, past participle of dire (“to say”), from Latin dīcō (“I say, I speak”). Not related to Italian dito (“finger”).

Comment (keszybz) [reply docs/PORTABLE_SERVICES.md]: Or are you saying that it's "dito" in German? That *is* confusing.

Comment (keszybz) [reply docs/PORTABLE_SERVICES.md]: Reminds me how culinary establishments in Poland (and probably elsewhere), serve "pene" instead of "penne".

Comment (poettering) [reply docs/PORTABLE_SERVICES.md]: yupp, in german it's "dito"

**Comment by poettering** [comment]
force pushed with the suggested changes. setting green label, since hcnages are so minimal.

**Pull request merged**